backend/src/services/tests/artist.test.ts

```diff
@@ -0,0 +1,91 @@
+import { database } from "../../config/database";
+import { Artist } from "../../models/artist.model";
+import { ArtistService } from "../artist.service";
+
+beforeEach(async () => {
+  await Artist.sync({ force: true });
+});
+
+test("create artist", async () => {
+  expect.assertions(3);
+  const artistService: ArtistService = new ArtistService();
+  const params = {
+    name: "testartist",
+  };
+  const create = await artistService.create(params);
+  expect(create.success).toEqual(true);
+  expect(create.body!.id).toEqual(1);
+  expect(create.body!.name).toEqual("testartist");
+});
+
+// create() does not accept params if name is missing
+
+test("show valid artist", async () => {
+  expect.assertions(3);
+  const artistService: ArtistService = new ArtistService();
+  const params = {
+    name: "testartist",
+  };
+  const create = await artistService.create(params);
+  if (!create.success) {
+    fail("Creating artist failed.");
+  }
+  const artist = create.body!;
+
+  const show = await artistService.show(artist.id);
+  expect(show.success).toEqual(true);
+  expect(show.artist!.id).toEqual(artist.id);
+  expect(show.artist!.name).toEqual(artist.name);
+});
+
+test("show invalid artist", async () => {
+  expect.assertions(1);
+  const artistService: ArtistService = new ArtistService();
+  await artistService
+    .show(1)
+    .then(() => fail())
+    .catch((err: Error) => expect(true).toEqual(true));
```

Comment:
I wonder if there is a better way to pass a test rather than expect(true).toEqual(true)? Not big deal either way I see what youre going for